deploy-edgeX.sh

```diff
@@ -31,13 +31,14 @@ run_service () {
 }
 
 run_service volume
+
 run_service consul
 
 run_service config-seed
 
-
-
-if [ "$SECURITY_SERVICE_NEEDED" = "true" ]; then
+# Security is enabled by default unless specified. This matches the rest of
+# the EdgeX system.
+if [ "$EDGEX_SECURITY_SECRET_STORE" != "false" ]; then
```

Comment:
I updated this to only use the `EDGEX_SECURITY_SECRET_STORE` environment variable. This makes it so that we only have to specify the one environment variable rather than two. Also, this makes the setup consistent with the rest of EdgeX.

Reply:
Ignore this comment. This has been reverted.